### ISSUE TITLE
Never clear values from _USED_DISPLAY_NR_LIST 

### ISSUE THREAD

**Issue by coderforlife** [ponty/PyVirtualDisplay#83]
When a display is stopped, it isn't removed from `_USED_DISPLAY_NR_LIST`. If displays are continuously started and stopped, this will eventually become impossible to get displays from.

(I have a server that is rendering things on-demand with workers that last a long time).

**Comment by ponty** [comment]
Do you run it on macOS? If yes, what is your XQuartz version?

**Comment by coderforlife** [comment]
No, on Linux. But I am ultimately created new displays fairly regularly and
the services are running for months before restarting.

**Comment by ponty** [comment]
I fixed it, but can you help with your setup? Distro version, Xvfb -help, Xvfb version.
`_USED_DISPLAY_NR_LIST` is only used when `-displayfd` command line option is not available 
which was added to Xvfb in 2012.
Do you have a very old system? How is it possible that you don't have `-displayfd`? 

Additional info from README:

"Recent X servers as of version 1.13 (Xvfb, too) support the -displayfd command line option: It will make the X server choose the display itself" https://stackoverflow.com/questions/2520704/find-a-free-x11-display-number/

Version 1.13 was released in 2012: https://www.x.org/releases/individual/xserver/
